src/io/flutter/editor/IndentsPass.java

```diff
@@ -3,7 +3,7 @@
 /*
  * @author max
  */
-package com.intellij.codeInsight.daemon.impl;
+package io.flutter.editor;
```

Comment:
Sounds good. How do I do that?

Reply:
When you see a Java analysis lint that we want to ignore, option-enter on it will bring up a menu to address the lint; a sub-menu will give you like 6 different ways to ignore it.

I think we have a few similar ignores checked into the codebase.